api/src/services/platform-service.ts

```diff
@@ -324,8 +325,19 @@ export class PlatformService extends DBService {
 
     const survey = await surveyService.getSurveyData(surveyId);
     const { surveyObservations } = await observationService.getSurveyObservationsWithSupplementaryData(surveyId);
-
-    const surveyDataPackage = new PostSurveyToBiohubObject(survey, surveyObservations, additionalInformation);
+    const surveyLocation = await surveyService.getSurveyLocationsData(surveyId);
+
+    const geometryFeatureCollection = {
+      type: 'FeatureCollection',
+      features: surveyLocation.flatMap((location) => (location.geojson as unknown) as Feature)
+    } as GeoJSON.FeatureCollection;
```

Comment:
You should be able to remove this cast, and just make it the type of `geometryFeatureCollection`?
It might just require you add an empty `properties: {}` field to comply.